### PR TITLE
Readme update & example app

### PR DESCRIPTION
**Pull request by uerceg**
- Updating README with universal links and device ID info.
- Adding example Cordova app to repository.

### CODE REVIEW

**Comment by uerceg** [comment]
@adjust/integration

File: README.md [hunk 1]
```diff
@@ -104,8 +104,9 @@ adjustConfig.setLogLevel(AdjustConfig.LogLevelAssert);    // disable errors as w
 ### 4. Google Play Services
 
 Since the 1st of August of 2014, apps in the Google Play Store must use the 
-[Google Advertising ID][google_ad_id] to uniquely identify the devices. To allow the adjust SDK 
-to use the Google Advertising ID, you must integrate the [Google Play Services][google_play_services].
+[Google Advertising ID][google_ad_id] to uniquely identify the devices. To allow
```
Comment (luckey01) [inline README.md]: to uniquely identify each device.

File: README.md [hunk 2]
```diff
@@ -104,8 +104,9 @@ adjustConfig.setLogLevel(AdjustConfig.LogLevelAssert);    // disable errors as w
 ### 4. Google Play Services
 
 Since the 1st of August of 2014, apps in the Google Play Store must use the 
-[Google Advertising ID][google_ad_id] to uniquely identify the devices. To allow the adjust SDK 
-to use the Google Advertising ID, you must integrate the [Google Play Services][google_play_services].
+[Google Advertising ID][google_ad_id] to uniquely identify the devices. To allow 
+the adjust SDK to use the Google Advertising ID, you must integrate the
```
Comment (luckey01) [inline README.md]: you must integrate Google Play Services.

File: README.md [hunk 3]
```diff
@@ -117,17 +118,17 @@ If you are using Proguard, add these lines to your Proguard file:
 -keep class com.google.android.gms.ads.identifier.** { *; }
 ```
 
-If you don't want to use Google Play Services in your app, you can remove them by editing `plugin.xml`
-file of the adjust SDK plugin. Go to `plugins/com.adjust.sdk` folder and open `plugin.xml` file.
-As part of the `<platform name="android">`, you can find following line which adds Google Play Services
-dependency:
+If you don't want to use Google Play Services in your app, you can remove them
```
Comment (luckey01) [inline README.md]: you can remove it

File: README.md [hunk 4]
```diff
@@ -117,17 +118,17 @@ If you are using Proguard, add these lines to your Proguard file:
 -keep class com.google.android.gms.ads.identifier.** { *; }
 ```
 
-If you don't want to use Google Play Services in your app, you can remove them by editing `plugin.xml`
-file of the adjust SDK plugin. Go to `plugins/com.adjust.sdk` folder and open `plugin.xml` file.
-As part of the `<platform name="android">`, you can find following line which adds Google Play Services
-dependency:
+If you don't want to use Google Play Services in your app, you can remove them 
+by editing `plugin.xml` file of the adjust SDK plugin. Go to `plugins/com.adjust.sdk`
```
Comment (luckey01) [inline README.md]: by editing the `plugin.xml`

File: README.md [hunk 5]
```diff
@@ -117,17 +118,17 @@ If you are using Proguard, add these lines to your Proguard file:
 -keep class com.google.android.gms.ads.identifier.** { *; }
 ```
 
-If you don't want to use Google Play Services in your app, you can remove them by editing `plugin.xml`
-file of the adjust SDK plugin. Go to `plugins/com.adjust.sdk` folder and open `plugin.xml` file.
-As part of the `<platform name="android">`, you can find following line which adds Google Play Services
-dependency:
+If you don't want to use Google Play Services in your app, you can remove them 
+by editing `plugin.xml` file of the adjust SDK plugin. Go to `plugins/com.adjust.sdk` 
+folder and open `plugin.xml` file. As part of the `<platform name="android">`, you
```
Comment (luckey01) [inline README.md]: open the

File: README.md [hunk 6]
```diff
@@ -117,17 +118,17 @@ If you are using Proguard, add these lines to your Proguard file:
...
 ```
 
-If you don't want to use Google Play Services in your app, you can remove them by editing `plugin.xml`
-file of the adjust SDK plugin. Go to `plugins/com.adjust.sdk` folder and open `plugin.xml` file.
-As part of the `<platform name="android">`, you can find following line which adds Google Play Services
-dependency:
+If you don't want to use Google Play Services in your app, you can remove them 
+by editing `plugin.xml` file of the adjust SDK plugin. Go to `plugins/com.adjust.sdk` 
+folder and open `plugin.xml` file. As part of the `<platform name="android">`, you 
+can find following line which adds Google Play Services dependency:
```
Comment (luckey01) [inline README.md]: find the following

**Comment by luckey01** [comment]
sorry, all the comments i just put in the commit 'luckey edit'

**Pull request merged**